app/app/settings/base.py

```diff
@@ -129,7 +129,7 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_ROOT = path.join(BASE_DIR, 'static')
-STATIC_URL = '/static/'
+STATIC_URL = environ.get("STATIC_URL", '/static/')
 MEDIA_ROOT = path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
```

Comment:
Even though we do not have `media`  but do not you think it is better to do the same with the media , at least for consistency and may be for future use? 

Also, For static or media, we need to add those variable to the template.env file

Reply:
Thanks @farassadek 

Re media, I don't think we need to bother with that. We will not have media in this application. In fact, I am going to comment those out and add a comment as to why. 

I've added the `STATIC_URL` variable to the template.env file.